bin/p2-fetch-manifest/main.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/square/p2/pkg/logging"
+	"github.com/square/p2/pkg/manifest"
+	"github.com/square/p2/pkg/store/consul"
+	"github.com/square/p2/pkg/store/consul/consulutil"
+	"github.com/square/p2/pkg/store/consul/flags"
+	"github.com/square/p2/pkg/version"
+	kingpin "gopkg.in/alecthomas/kingpin.v2"
+)
+
+const (
+	cmdFetchText = "fetch"
+)
+
+var (
+	cmdFetch     = kingpin.Command(cmdFetchText, "Fetch the manifest for a given pod from the datastore and write it to a file on the disk. Combines well with p2-launch.")
+	fetchTree    = cmdFetch.Flag("tree", "Intent or Reality").Default("intent").Short('t').Enum("intent", "reality")
+	fetchPod     = cmdFetch.Flag("podID", "The pod ID for which to fetch its manifest").Short('i').Required().String()
+	fetchEntropy = cmdFetch.Flag("force-restart", "Introduce entropy into the manifest before outputting it. Scheduling the resulting manifest will definitely cause a restart of the pod.").Short('p').Bool()
```

Comment:
From a purely functional view, changing a manifest to get it to restart is an orthogonal and unrelated task to fetching a single manifest. They are independently useful on their own. It doesn't make sense to me to couple them together into the same binary, much less exposing one as a mere flag.

Could you make "force restart" into its own binary? If that strikes you as too heavyweight, is there another existing P2 command that locally modifies a manifest? Putting the "force restart" modification into that command would at least group similar kinds of functionality.

nit: Your use of "entropy" here does not fit its common definition in computer science. I also don't think it's relevant _how_ a restart is forced, merely that it is.